### PR TITLE
Update a few python ports

### PR DESCRIPTION
**Pull request by catap**
#### Description

I'm a maintainer or co-maintainer of all this ports.

It is simple and clean update.

###### Type(s)

- [x] enhancement

###### Tested on

macOS 12.6.3 21G419 x86_64
Xcode 14.2 14C18

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-pathvalidate/Portfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-pathvalidate
-version             2.5.2
+version             3.0.0
```
Comment (raimue) [inline python/py-pathvalidate/Portfile:7]: I didn't check, but I guess this will no longer be API compatible. Do we have dependents?

**Review by raimue — COMMENTED** [review]
Update to osc looks good. Thanks for the port update! :+1:

File: python/py-pathvalidate/Portfile (hunk 1 above)
Comment (catap) [reply python/py-pathvalidate/Portfile:7]: @raimue only one port `qobuz-dl` and it works :)

Comment (catap) [reply python/py-pathvalidate/Portfile:7]: BTW, based on https://github.com/thombashi/pathvalidate/releases/tag/v3.0.0 changes aren't that huge and should be safe

**Pull request merged**